requirements.txt

```diff
@@ -4,6 +4,7 @@ azure-mgmt-storage==20.0.0
 azure-mgmt-resource==21.1.0
 azure-storage-blob==12.13.0
 azure-identity==1.10.0
+cachetools
```

Comment:
where we using this ?

Reply:
> where we using this ?

We use the `cached` decorator in `ocw.lib.openqa.get_url()` to cache the results.